documentation/practices.md

```diff
@@ -80,5 +80,37 @@ Configuration properties for content URLs are:
+
+The Indi component available is a stand-alone component created solely to fetch the `indi-embed` libraries, then display the corresponding elements that were passed into the component as a structured list.
+
+The required parameters of this component are as follows:
+* `render`: A list of the elements to display from Indi. Example: `render={['carousel', 'brand', 'product']}`. Note the correct configurations must be provided for the element to render correctly.
+* `configuration`: The structured object consisting of the configurations to provide to the Indi components. These are defined in `ep.config.json` in the `indi` element.
+* `keywords`: A string of the keywords for the product in which the Indi component is displayed for.
+
+Static strings for the Indi component have been localized in the Reference Storefront and are overrided on a per-usage basis for the Indi component. This will ensure the component displays the correct text in the storefront based on the locale which a customer chooses during their shopping flow.
+
+Configuration properties for Indi component's requirements are:
+* `indi.enable`: Enable the integration component for Indi. More information may be found available here [https://indi.com/](https://indi.com/).
+* `indi.carousel`: Configurations for the Indi carousel component.
+* `indi.carousel.apikey`: The apikey used to connect the carousel to Indi.
+* `indi.carousel.id`: The apikey used to connect the carousel to Indi.
+* `indi.carousel.size`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.theme`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.round_corners`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.show_title`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.show_views`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.show_likes`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.show_buzz`: Used when theming the carousel displaying content from Indi.
+* `indi.carousel.animate`: Used when theming the carousel displaying content from Indi.
+* `indi.productReview`: Configurations for the Indi product review component.
+* `indi.productReview.submit_button_url`: The URL provided by Indi which directs to the action performed when the submit button is clicked on a product review.
+* `indi.productReview.thumbnail_url`: The thumbnail provided by Indi for the submit button.
+* `indi.brandAmbassador`: Configurations for the Indi brand ambassador component.
+* `indi.brandAmbassador.submit_button_url`: The URL provided by Indi which directs to the action performed when the submit button is clicked on a brand ambassador signup.
+* `indi.brandAmbassador.thumbnail_url`: The thumbnail provided by Indi for the submit button.
```

Comment:
Can we have a table with the following details:

Functionality|     Parameter |            Description |  >> **Table header**

**Integration**

**Theming**          |  List all parameters from (101 to 108)|    Description for 101
                                                                                                    ...


**Product View** .  |  (line 109 - 111)|

**Brand ambassador** |(line 112-114) |

Reply:
Adding in latest commit